Assets/VRTK/Scripts/VRTK_InteractTouch.cs

```diff
@@ -401,6 +401,7 @@ private void CreateTouchCollider()
                 {
                     controllerCollisionDetector = Instantiate(customRigidbodyObject, transform.position, transform.rotation) as GameObject;
                     controllerCollisionDetector.transform.SetParent(transform);
+                    controllerCollisionDetector.transform.localScale = transform.localScale;
```

Comment:
I believe once you set the parent it will try and rescale it down to match. This is why the controllers scale that was fixed 2 days ago broke.

Maybe it would be safer to store the localScale before setting the parent, and then setting it to that. What do you think?